Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-alpine
+
+# Add mozci source code and setup instructions
+WORKDIR /src
+COPY LICENSE  poetry.lock  pyproject.toml  README.md ./
+COPY mozci /src/mozci/
+
+# Run in a single step to get a small image
+RUN apk add --virtual build gcc libffi-dev musl-dev postgresql-dev && \
+  # Setup latest poetry through pip
+  pip install poetry && \
```

Comment:
Nit: add `--no-cache-dir` to avoid the pip cache being included in the image

Reply:
Done :+1: